src/Toolkit/F.php

```diff
@@ -105,8 +108,35 @@ class F
         ],
     ];
 
+    /**
```

Comment:
Since `$resizable` and `$viewable` are directly connected (and only used by) `Kirby\Image|Image`, I think they could make more sense in that class than in `F`. But curious to hear other opinions first before changing this in your PR already. @lukasbestle maybe?

Reply:
I agree with Nico.

Reply:
Thank you both for reviewing 👍 I'll implement after final review.

Reply:
Done